go.mod

```diff
@@ -1,6 +1,8 @@
 module terraform-provider-genesyscloud
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
 
```

Comment:
This usually happens if you're running a go version other than 1.20 on your machine. If you switch to go 1.20 and re-run `go mod tidy` , this shouldn't happen. We're avoiding upgrading from go1.20 at the moment because it causes problems with the tool we use to release new versions of the provider